docs/source/releases/v1_11_7a0.rst

```diff
@@ -0,0 +1,55 @@
+ | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+ | # # #
+ | # # # Author:
+ | # # # Naval Research Laboratory, Marine Meteorology Division
+ | # # #
+ | # # # This program is free software: you can redistribute it and/or modify it under
+ | # # # the terms of the NRLMMD License included with this program. This program is
+ | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+ | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+ | # # # for more details. If you did not receive the license, for more information see:
+ | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Version 1.11.7a0 (2023-10-06)
+*****************************
+
+* Update code to adhere to newly named geostationary sectors
+
+Refactoring Updates
+-------------------
+
+Update code for newly named Geostationary sectors
+-------------------------------------------------
+
+*From issue GEOIPS#304: 2023-08-27, Rename Geostationary Sectors*
+
+GeoIPS has a branch which renamed geostationary sectors. To adhere to these newly named
+sectors, we need to update scripts in this package, and well as the outputs, to use
+these newly named sectors.
+
+::
+
+    modified: tests/outputs/abi.Cloud-Fraction.imagery_clean/20230411.160200.goes-16.clavrx.Cloud-Fraction.goes16.71p70.cira.10p0.png
+    modified: tests/outputs/ahi.Cloud-Base-Height.imagery_clean/20230411.030000.him9.clavrx.Cloud-Base-Height.himawari8.47p95.cira.10p0.png
+    modified: tests/outputs/ahi.Cloud-Fraction.imagery_clean/20230411.030000.him9.clavrx.Cloud-Fraction.himawari8.71p70.cira.10p0.png
+    modified: tests/outputs/ahi.Cloud-Mask.imagery_clean/20230411.030000.him9.clavrx.Cloud-Mask.himawari8.71p70.cira.10p0.png
+    modified: tests/outputs/ahi.Cloud-Optical-Depth.imagery_clean/20230411.030000.him9.clavrx.Cloud-Optical-Depth.himawari8.69p44.cira.10p0.png
+    modified: tests/outputs/ahi.Cloud-Phase.imagery_clean/20230411.030000.him9.clavrx.Cloud-Phase.himawari8.33p41.cira.10p0.png
+    modified: tests/outputs/ahi.Cloud-Temp-ACHA.imagery_clean/20230411.030000.him9.clavrx.Cloud-Temp-ACHA.himawari8.48p00.cira.10p0.png
+    modified: tests/outputs/ahi.Cloud-Top-Height.imagery_clean/20230411.030000.him9.clavrx.Cloud-Top-Height.himawari8.48p00.cira.10p0.png
+    modified: tests/outputs/ahi.Cloud-Type.imagery_clean/20230411.030000.him9.clavrx.Cloud-Type.himawari8.71p70.cira.10p0.png
+    modified: tests/outputs/ahi.Effective-Radius.imagery_clean/20230411.030000.him9.clavrx.Effective-Radius.himawari8.48p00.cira.10p0.png
+    modified: tests/outputs/ahi.Temp-11p0.imagery_clean/20230411.030000.him9.clavrx.Temp-11p0.himawari8.71p70.cira.10p0.png
+    modified: tests/outputs/ahi.Temp-3p75.imagery_clean/20230411.030000.him9.clavrx.Temp-3p75.himawari8.71p70.cira.10p0.png
```

Comment:
Do we know if these images actually have differences, or they were just moved?  We should probably make sure they do not actually have any differences before merging this (since there should NOT be any changes.  If there are any changes at all, something got screwed up ?)

Reply:
I just rolled them all back - now they are renamed and uncnagned.  It was matplotlib 3.7.1 vs 3.7.2 in the headers of the images that were different.